packages/RichTextEditing-Tests.package/RichTextTextStructureTests.class/instance/testListTextStructureIsNotAligned.st

```diff
@@ -0,0 +1,4 @@
+unit-tests
+testListTextStructureIsNotAligned
```

Comment:
Perhaps `IsNotAlignment` instead of `IsNotAligned`? Since we are not testing the rendering here